src/pkg/clients/core/chartv2.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/goharbor/harbor/src/chartserver"
 )
 
-func (c *client) ListAllCharts(project, repository string) ([]*chartserver.ChartVersion, error) {
+func (c *clientv2) ListAllCharts(project, repository string) ([]*chartserver.ChartVersion, error) {
 	url := c.buildURL(fmt.Sprintf("/api/chartrepo/%s/charts/%s", project, repository))
```

Comment:
The chart related APIs are also bumped up to v2.0 already

Reply:
OK, will fix this in another PR